extension-inspector-packaging/src/main/resources/application.yaml

```diff
@@ -2,6 +2,7 @@
 spring:
   main:
     banner_mode: "off"
+    allow-circular-references: true
```

Comment:
Why do we need allow-circular-reference to be true

Reply:
Removed the circular reference properties used in yaml